PostDeploymentActions/updateUpdaterTimer.ps1

```diff
@@ -0,0 +1,9 @@
+$randH = Get-Random -minimum 0 -maximum 11
+$randM = Get-Random -minimum 0 -maximum 59
+$randS = Get-Random -minimum 0 -maximum 59
+$randH12 = $randH + 12
+$new_schedule = "$randS $randM $randH,$randH12 * * *"
```

Comment:
btw just to confirm - will post deployment actions be updated when we release new version of azure-collector (by merging something to the master branch)? 
I'm asking in case if we would like to change this logic somehow to the already deployed functions in the future

Reply:
yes, it will. Post deployment action is called every time web app is synced with github repo.